server/src/main/java/io/deephaven/server/table/ops/ApplyPreviewColumnsGrpcImpl.java

```diff
@@ -23,7 +25,10 @@ protected ApplyPreviewColumnsGrpcImpl() {
     }
 
     @Override
-    public Table create(ApplyPreviewColumnsRequest request, List<SessionState.ExportObject<Table>> sourceTables) {
+    public Table create(final AuthContext authContext,
+            final ApplyPreviewColumnsRequest request,
+            final List<SessionState.ExportObject<Table>> sourceTables) {
+        authContext.requirePrivilege(TableServicePrivilege.CAN_APPLY_PREVIEW_COLUMNS);
```

Comment:
I wonder what a user is supposed to do if this fails. Should we drop the columns that would be previewed?